src/test/java/pages/InternetHerokuappPage.yml

```diff
@@ -0,0 +1,10 @@
+urls:
+   default: http://the-internet.herokuapp.com/
```

Comment:
I believe this file already exists. You should just be adding a new element to it.

Reply:
I don't see any duplicate in my local.

Reply:
src/test/java/com.herokuapp/InternetPage.yml

This file was not created to the correct specs. It should be in a package matching the production URL.

Reply:
This still needs to be deleted and the correct page used.